docs/guides/Glossary.md

```diff
@@ -1,13 +1,20 @@
-# Glossary
+## Glossary
 
-## Implicit cast
+### Implicit cast
 
 A *implicit cast* is a transformation done automatically (implicitly) by one CARTO VL expression to another one. Its purpose is to provide *syntax sugar* over another expression which is equivalent but more verbose and difficult.
 
 For example, when using `ramp($numericProperty, [red, blue])` the `ramp` expression will implicitly cast `$numericProperty` to `linear($numericProperty)`, and the `linear` expression will implicitly cast that to `linear($numericProperty, globalMin($numericProperty), globalMax($numericProperty))`, which is a much larger and more verbose form, but it is semantically the same.
 
 
-## Variables
+### Others bucket
+
+The *others bucket* is a category that is generated by some CARTO VL functions like `buckets` and `top`. This bucket will contain all the categories that were grouped together in the last place.
+
+There is more information in the [Data driven visualization guide]()
+
+
+### Variables
```

Comment:
I think we should use by convention `JavaScript` and `JavaScript API` instead of `Javascript`.

Reply:
Where exactly? :confused: